crypto/doc/nrf_cc310_mbedcrypto.rst

```diff
@@ -186,8 +180,9 @@ An alternative to allocating this on the heap is to provide a reference to a sta
 Usage restrictions
 ------------------
 
-On the nRF9160 SiP, the nrf_cc3xx_mbedcrypto library is restricted to only work in secure processing environment.
-The library uses mutexes to ensure single usage of hardware modules.
+The library can not be used in the non-secure domain of an application that uses ARM TrustZone.
+
+The hardware can only process one request at a time, therefore this library uses mutexes to ensure that multiple threads can use it in a thread-safe way.
```

Comment:
@ Line 185, break this into two sentences. 
"The hardware can only process one request at a time. Therefore, this library uses mutexes to ensure that multiple threads can use it in a thread-safe way."

Reply:
much better